syndeo-template/src/main/java/com/google/cloud/syndeo/WritePipelineSpecForTesting.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.syndeo;
+
+import static org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.MoreObjects.firstNonNull;
+
+import com.google.cloud.syndeo.common.ProviderUtil.TransformSpec;
+import com.google.cloud.syndeo.v1.SyndeoV1.PipelineDescription;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.beam.sdk.schemas.Schema;
+import org.apache.beam.sdk.schemas.Schema.Field;
+import org.apache.beam.sdk.schemas.Schema.FieldType;
+
+public class WritePipelineSpecForTesting {
+
+  public static final String GCP_PROJECT =
```

Comment:
From the looks of the README, we're passing this in on the command line rather than trying to read the environment variable. Shouldn't this be `getProperty` rather than `getenv`?

Reply:
I've not set this up for all executables. This particular one has everything hardcoded or passed via env variables (`mvn compile exec:java -Dexec.mainClass="com.google.cloud.syndeo.WritePipelineSpecForTesting"`) - but the `SyndeoTemplate` one does not yet. I can create a follow-up work item to document / fix this. WDYT?

Reply:
Sure.